src/rustdoc-json-types/lib.rs

```diff
@@ -82,10 +97,13 @@ pub struct Item {
     pub links: FxHashMap<String, Id>,
     /// Stringified versions of the attributes on this item (e.g. `"#[inline]"`)
     pub attrs: Vec<String>,
+    /// Information about the Item’s deprecation, if present.
```

Comment:
NIT: If we're not linking/using `code formatting`, I think caps here is kind of jarring.

```suggestion
    /// Information about the item’s deprecation, if present.
```